pkg/kubelet/container/sync_result.go

```diff
@@ -54,6 +54,7 @@ var (
 	ErrKillContainer    = errors.New("KillContainerError")
 	ErrVerifyNonRoot    = errors.New("VerifyNonRootError")
 	ErrRunInitContainer = errors.New("RunInitContainerError")
+	ErrKillPodSandbox   = errors.New("KillPodSandboxError")
```

Comment:
@feiskyer The errors and sync results are a bit annoying. :P 
However, since you've already added `ErrKillPodSandbox`, why don't we add `ErrCreatePodSandbox`?

It seems that you just use `err` and `fmt.Sprintf("CreatePodSandbox failed: %v", err)`. https://github.com/kubernetes/kubernetes/pull/28396/commits/6e8f82db7e2c7057544b5dd8bdc9356900bc68ad#diff-8da51ab012ecbdf2b013ecbcf1c37033R406